Projects/App/Sources/UI/Review/Views/EvaluationButton.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  EvaluationButton.swift
+//  App
+//
+//  Created by 리아 on 2022/10/20.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class EvaluationButton: UIView {
+    
+    enum EvaluationType {
+        case good
+        case soso
+        case bad
+        
+        var image: UIImage? {
+            switch self {
+            case .good:
+                return UIImage(.img_reviewfriends_good)
+            case .soso:
+                return UIImage(.img_reviewfriends_soso)
+            case .bad:
+                return UIImage(.img_reviewfriends_bad)
+            }
+        }
+        
+        var desc: String {
+            switch self {
+            case .good:
+                return "맛집이요"
+            case .soso:
+                return "무난했어요"
+            case .bad:
+                return "별로였어요"
+            }
+        }
+    }
+    
+    // MARK: - Properties
+    private var cancelBag = Set<AnyCancellable>()
+    @Published var isSelected = false
+
+    private let backgroundView = UIView()
+    private let evaluationStackView = UIStackView()
+    private let evaluationIcon = UIImageView()
+    private let evaluationLabel = UILabel()
+    public let button = UIButton()
+    
+    // MARK: - LifeCycle
+    
+    public init(type: EvaluationType) {
+        super.init(frame: .zero)
+        configureUI(type: type)
+        createLayout()
+        bind()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+}
+
+// MARK: - Binding
+
+extension EvaluationButton {
+        
+    private func bind() {
+        $isSelected
+            .sink { [weak self] isSelected in
+                guard let self = self else { return }
+                self.backgroundView.backgroundColor = isSelected ? .black : .zestyColor(.grayF6)
+                self.evaluationLabel.textColor = isSelected ? .white : .darkGray
+            }
+            .store(in: &cancelBag)
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension EvaluationButton {
+    
+    private func configureUI(type: EvaluationType) {
+        evaluationIcon.image = type.image
```

Comment:
p1;

아이콘이 UIImage로 들어가기 때문에 contentMode를 지정해줘야 합니다!
기본으로 .scaleToFit으로 되어있는것 같아서 비율이 깨질 수도 있네요.

```suggestion
        evaluationIcon.image = type.image
        evaluationIcon.contentMode = .scaleAspectFit
```

Reply:
아 오토레이아웃으로 비율 고정해놔서 상관 없을 줄 알았는데
나중에 이미지 어셋이 바뀔 수도 있으니까 해두는 게 좋겠네요!

Reply:
9a5214929a64a8203485870f4a8d812d2989f655 반영 완료요~